ethportal-peertest/src/events.rs

```diff
@@ -1,10 +1,12 @@
 use discv5::{Discv5Event, TalkRequest};
+use hex;
 use log::{debug, error, warn};
 use std::convert::TryInto;
+use std::str::FromStr;
 use std::sync::Arc;
 use tokio::sync::mpsc;
 use trin_core::portalnet::overlay::OverlayProtocol;
-use trin_core::portalnet::types::Message;
+use trin_core::portalnet::types::{Message, ProtocolId};
 use trin_core::portalnet::utp::UtpListener;
 
 pub const HISTORY_NETWORK: &str = "history";
```

Comment:
If we are now using ProtocolId these constants for history, state, and utp should be removed since uTP is being used instead now.

Reply:
Right, good catch! I'm curious why clippy doesn't yell at me.